spring-cloud-deployer-resource-support/src/main/java/org/springframework/cloud/deployer/resource/support/DelegatingResourceLoader.java

```diff
@@ -159,7 +159,7 @@ private static boolean existsAsFile(Resource resource) {
 			resource.getFile();
 			return true;
 		}
-		catch (IOException e) {
+		catch (Exception e) {
```

Comment:
maybe not since this is only called internally anyways... implementation detail

will merge